@xen-orchestra/lite/src/components/host/HostTabBar.vue

```diff
@@ -16,7 +16,7 @@
       {{ t('tasks') }}
     </RouterTab>
     <RouterTab :to="{ name: 'host.vms', params: { uuid } }" disabled>
-      {{ t('vms') }}
+      {{ t('vms', 0) }}
```

Comment:
According to Figma, show the vms variant.
```suggestion
      {{ t('vms', 2) }}
```